lib/modules/onc_program/onc_parameters.rb

```diff
@@ -15,6 +15,11 @@ class TestingInstance
       property :onc_sl_patient_id, String
       property :onc_sl_oauth_token_endpoint, String
 
+      property :onc_public_confidential_client, Boolean
+      property :onc_public_client_id, String
+      property :onc_public_client_secret, String
```

Comment:
What is this used for?  I don't see it as a field in the "run" modal and it doesn't look like it's used anywhere.

Reply:
You are correct, we just need the client_id.  I added those because i thought they would have to exist to have the sequence work, but then realized they didn't have to (but then forgot to remove them here).  Removing...